jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -191,12 +191,13 @@ def step030_customize_settings(script_dir, puppet_settings) {
     }
 }
 
-def step035_customize_hocon_settings(script_dir, settings) {
+def step035_customize_hocon_settings(script_dir, settings, server_era) {
     if (settings == null) {
         echo "Skipping hocon settings customization; no overrides found in `hocon_settings`."
     } else {
         settings_json = JsonOutput.toJson(settings)
-        withEnv(["PUPPET_GATLING_HOCON_SETTINGS=${settings_json}"]) {
+        withEnv(["PUPPET_GATLING_HOCON_SETTINGS=${settings_json}",
+                 "PUPPET_SERVER_SERVICE_NAME=${server_era["service_name"]}"]) {
```

Comment:
OK, that makes  sense, it just looked like this change came in in your latest commit, and it seemed like it would have been needed prior to that.  Might have just been some screwiness  in the github diff though.  I agree that var is necessary for the restart step.